### PR TITLE
docs: Add known limitations

### PR DESCRIPTION
**Pull request by MaxymVlasov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,6 +286,10 @@ Check the [source file](https://github.com/antonbabenko/pre-commit-terraform/blo
 
 ## Hooks usage notes and examples
 
+### Known limitations
+
+Terraform operate on per-dir basic, when `pre-commit` only supports files and files that exist. That means if you only remove tf-related file without any other changes in same dir, check will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048)
```
Comment (MaxymVlasov) [inline README.md:291]:
```suggestion
Terraform operates on per-dir basis, when `pre-commit` only supports files and files that exist. That means if you only remove the tf-related file without any other changes in the same dir, checks will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048)
```

File: README.md [hunk 2]
```diff
@@ -285,6 +286,10 @@ Check the [source file](https://github.com/antonbabenko/pre-commit-terraform/blo
 
 ## Hooks usage notes and examples
 
+### Known limitations
+
+Terraform operates on per-dir basis, when `pre-commit` only supports files and files that exist. That means if you only remove the tf-related file without any other changes in the same dir, checks will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048)
```
Comment (MaxymVlasov) [inline README.md:291]:
```suggestion
Terraform operates on a per-dir basis, when `pre-commit` only supports files and files that exist. That means if you only remove the tf-related file without any other changes in the same dir, checks will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048).
```

File: README.md [hunk 3]
```diff
@@ -285,6 +286,10 @@ Check the [source file](https://github.com/antonbabenko/pre-commit-terraform/blo
 
 ## Hooks usage notes and examples
 
+### Known limitations
+
+Terraform operates on a per-dir basis, when `pre-commit` only supports files and files that exist. That means if you only remove the tf-related file without any other changes in the same dir, checks will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048).
```
Comment (yermulnik) [inline README.md:291]:
```suggestion
Terraform operates on a per-dir basis, while `pre-commit` framework only supports files and files that exist. This means if you only remove the TF-related file without any other changes in the same dir, checks will be skipped. Example and details [here](https://github.com/pre-commit/pre-commit/issues/3048).
```

**Review by yermulnik — APPROVED** [review]

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 1.83.6 :tada:

**Pull request merged**